src/compiler/parse/read/style.ts

```diff
@@ -16,6 +16,12 @@ export default function read_style(parser: Parser, start: number, attributes: No
 
 	const content_end = parser.index;
 
+	// discard styles when css is disabled
+	if (parser.css_mode === 'none') {
+		parser.read(/<\/style\s*>/);
```

Comment:
@Prinzhorn suggested in https://github.com/sveltejs/svelte/pull/7890#discussion_r983677453 to extract this to a constant used on lines 11, 21, and 76. I think that's a good idea

Reply:
Sounds good. Should be ok now. Thanks